contracts/token/minter/OwnedTimedMinter.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity ^0.8.0;
+
+import "./FeiTimedMinter.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/// @title OwnedTimedMinter
+/// @notice A FeiTimedMinter that mints only when called by an owner
+contract OwnedTimedMinter is FeiTimedMinter, Ownable {
+
+    /**
+        @notice constructor for OwnedTimedMinter
+        @param _core the Core address to reference
+        @param _owner the minter and target to receive minted FEI
+        @param _frequency the frequency buybacks happen
+        @param _initialMintAmount the initial FEI amount to mint
+    */
+    constructor(
+        address _core,
+        address _owner,
+        uint256 _frequency,
+        uint256 _initialMintAmount
+    ) 
+        FeiTimedMinter(_core, _owner, 0, _frequency, _initialMintAmount)
+    {
+        transferOwnership(_owner);
+    }
+
+    /// @notice triggers a minting of FEI by owner
+    function mint() public override onlyOwner {
```

Comment:
Who will the owner of this contract be? I would assume it's either the guardian or the OA multisig?